public/app/features/teams/TeamList.tsx

```diff
@@ -155,7 +133,7 @@ export const TeamList = ({
         },
       },
     ],
-    [displayRolePicker, editorsCanAdmin, roleOptions, signedInUser, deleteTeam, enableSort]
```

Comment:
i think from the previous [commit](https://github.com/grafana/grafana/pull/75005/commits/706b89c2be72e226065851a00a466d6e7a2923ab), where we added back `editorsCanAdmin`, this part of the component was not added back with `editorsCanAdmin`.

Same with lines of the TeamList tests https://github.com/grafana/grafana/pull/75005/files#diff-e3849982f2e4819202742a5531d32dd59ec54c957627c1032cb7b1f7f9a9dc92L82

Could be incorrect here and this is moved somewhere else? 🤔 

This same comment applied to TeamPages.tsx, TeamPages.test.tsx

Reply:
We're not using `editorsCanAdmin` here anymore for permission evaluation - instead we use it to set the permissions correctly on the backend ([here](https://github.com/grafana/grafana/blob/main/pkg/api/accesscontrol.go#L240-L242)), so we only need to evaluate the RBAC permissions on the frontend.

That was the whole reason why I thought that we can remove `editorsCanAdmin` from the frontend altogether, as it's not used in the frontend code anymore. But I put it back in in case any of the plugins are relying on it.